opendebates/settings.py

```diff
@@ -8,6 +8,39 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 FIXTURE_DIRS = [os.path.join(BASE_DIR, 'fixtures')]
 
+SITE_ID = 1
+SITE_DOMAIN = os.environ.get("SITE_DOMAIN", "127.0.0.1:8000")
+SITE_DOMAIN_WITH_PROTOCOL = os.environ.get("SITE_PROTOCOL", "http://") + SITE_DOMAIN
+
+SITE_THEMES = {
+    'florida': {
+        "HASHTAG": "FLOpenDebate",
+        "HEADER_TITLE": _("WELCOME TO THE\nFLORIDA OPEN DEBATE"),
```

Comment:
For per-site copy changes like these, I'd been imagining that we could (ab)use the gettext/translations mechanism -- creating message catalogs for "florida" and "opemdemquestions" (`manage.py makemessages -l florida`) and managing any copy overrides in those django.po files, so that we don't have to accumulate a potentially big list of strings in python code and don't have to change the template or view files at all, as long as they're already i18n-aware.

Would that work here?  (So instead of this HEADER_TITLE and HEADER_COPY stuff, we'd just have `settings.LANGUAGE_CODE` depend on the active site theme and/or be set in local_settings.) Are there design or performance reasons why that would be problematic?  (In terms of performance, I *think* the benchmarks have all been done with i18n active already..)